website/src/components/Chat/ChatListBase.tsx

```diff
@@ -78,19 +80,22 @@ export const ChatListBase = memo(function ChatListBase({
       }}
       {...props}
     >
-      <CreateChatButton
-        py="5"
-        leftIcon={<Plus size="16px"></Plus>}
-        variant="outline"
-        justifyContent="start"
-        colorScheme="blue"
-        borderRadius="lg"
-        mx="3"
-        mb="2"
-        onUpdated={handleCreateChat}
-      >
-        {t("create_chat")}
-      </CreateChatButton>
+      <Flex>
+        <CreateChatButton
+          py="5"
+          leftIcon={<Plus size="16px" />}
+          variant="outline"
+          justifyContent="start"
+          colorScheme="blue"
+          borderRadius="lg"
+          mx="3"
+          onUpdated={handleCreateChat}
+          flexGrow="1"
+        >
+          {t("create_chat")}
+        </CreateChatButton>
+        {allowHiddenChats && <ChatHiddenSwitch onChange={setIncludeHidden} value={includeHidden} />}
```

Comment:
Can you make it a selection instead? So user can select visible chat, hidden chat, trainable chat, and maybe shared chat in the future

Reply:
Makes sense, added!